io.github.zen_browser.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.github.zen_browser
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: '23.08'
+command: launch-script.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pcsc
+  - --socket=cups
+  - --persist=.zen
+  - --filesystem=xdg-download:rw
+  - --device=all
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+modules:
+  - name: zen_browser
+    buildsystem: simple
+    build-commands:
+      - mv zen/* /app/bin/
+
+      - install -d /app/share/icons
+
+      - cp -r icons/* /app/share/icons
```

Comment:
```suggestion
      - install -Dm0644 ${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/${FLATPAK_ID}.png
```

Reply:
I removed the subfolders though, idk if it affects anything